app/ui.R

```diff
@@ -1,55 +1,82 @@
+  
   theme = "styles.css",
+  
   tabPanel( #at the top of every page to navigate through the entire dashboard, contains tabs for home
+  
     title = "Home",
     navlistPanel( 
        id = "tabset",
        tabPanel("Summary Data Profile", "Panel one contents"),
        tabPanel("Info Panel", "Panel two contents")
-           )),
+           )
+    ),
+  
   tabPanel( #at the top of every page to navigate through the entire dashboard, contains tabs for data quality
+    
     title = "Data Quality",
     navlistPanel(
       id = "tabset",
       tabPanel("Home", "Panel one contents"),
+      
       tabPanel("Completeness", "Panel two contents",
-               fluidRow(
-                   column(3,
-                          selectInput("hb_in", "Health Board", choices = c("(All)", unique(smr_completeness$hb_name)))
-                          ),
-                   column(3,
-                          selectInput("month_in", "Month", choices = c("(All)", unique(smr_completeness$month_record_inserted)))
+               
+              fluidRow(
+                column(4, 
+                       selectInput("smr_in", "SMR", choices = c("(All)", unique(smr_completeness$smr)))
                           ),
-                   column(3,
-                          selectInput("data_item_in", "Data Item", choices = c("(All)", unique(smr_completeness$data_item)))
+                column(4,
+                       selectInput("hb_in", "Health Board", choices = c("(All)", unique(smr_completeness$hb_name)))
                           ),
-                   column(3,
-                          selectInput("percentage_in", "Percentage Complete", choices = c("(All)","0% - 20%", "20% <", "50% <", "80% <", "100%" ))
-                   )
-                        ),
-               fluidRow(
-                 tableOutput("completeness_table")
-                      )
+                column(4,
+                       selectInput("data_item_in", "Data Item", choices = NULL)
+                          )
+                     ),
+              
+              fluidRow(
+                checkboxGroupInput("percentage_in", 
+                                   "Completeness percentage range",
+                                   choices = list("Above 60% complete" = 1,
+                                                  "Between 40% and 60% complete" = 2,
+                                                  "Below 40% complete" = 3),
+                                   selected = c(1,2,3),
+                                   inline = TRUE
+                                   )
+                    ),
+                      
+              
+              fluidRow(
+                #add icon colors
+                tags$style(".glyphicon-ok {color:#2BE532}
+                          .glyphicon-warning-sign {color:#FFC300}
+                          .glyphicon-flag {color:#C70039}"),
+                DT::dataTableOutput("completeness_table")
+                    )
```

Comment:
When using the `fluidRow` system, the columns need to add to 12. There are issues falling out of this with padding/spacing and things not showing as expected across window sizes.

Reply:
thanks, attempted to resolve the padding issues so that the tables, filters and side panels fall into place better :)